database/postgres/db_postgres_benchmark.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-console */
+const { Client } = require('pg');
+const config = require('./db_postgres_config');
+const { getRandomInt } = require('../data/data-generator-utils');
+
+const client = new Client(config);
+
+// config variables
+const records = 5048; // number of reviews
+const joinedQuery = true;
+const connectionType = 'client'; // client or pool
+const sameRecord = false; // whether the query asks for a different record each time
+
+let reviewID;
+let query;
+if (!sameRecord) {
+  const last90 = Math.floor(records * 0.9);
+  reviewID = getRandomInt(last90, records);
+} else {
+  reviewID = Math.floor(records * 0.9);
+}
+
+if (!joinedQuery) {
+  query = `SELECT * FROM reviews WHERE review_id=${reviewID}`;
```

Comment:
Directly manipulating query string with external variables poses security vulnerability. Instead, consider passing the values in an array as the second positional argument of the query method.